lib/shoryuken/util.rb

```diff
@@ -4,13 +4,13 @@ def logger
       Shoryuken.logger
     end
 
-    def fire_event(event, reverse = false)
+    def fire_event(event, options = {}, *event_args)
```

Comment:
Hi @hasbrettn 

I prefer `def fire_event(event, reverse = false, *event_args)` or `def fire_event(event, reverse = false, *event_options)` to avoid introducing breaking changes.